tests/apps/jsonlayerstest.cpp

```diff
@@ -396,6 +396,7 @@ static void init_frames(int32_t width, int32_t height) {
         case LAYER_TYPE_GL:
           renderer = new GLCubeLayerRenderer(gbm.dev, false);
           break;
+#ifdef USE_MINIGBM
```

Comment:
hmm why do we need this ? Lets avoid this and video layer will fail in this case which is fine since it tells us the current state of things.